### ISSUE TITLE
Complete guides for assets compilation 

### ISSUE THREAD

**Issue by eliasjpr** [amberframework/amberframework.org#45]
As a developer I would like to understand how asset compilation works within Amber

AC
- Read instructions and directions on how assets compilation work.
- Has code snippets examples within guide

**Comment by faustinoaq** [comment]
You're right, I working on an `amber watch` rewrite, so we should be able to use whatever front-end tool we want :tada: 

Ref: https://github.com/amberframework/amber/issues/476

**Comment by faustinoaq** [comment]
Also, the assets compilation depend on what tools are you using, so, amber will be agnostic to assets compilation: you can use `yarn`, `webpack`, `rollup`, `parcel` and the generated assets will depend on how you use these tools that you already know. A very basic assets management will be available, though (`cp` and `cat`) :wink:

**Comment by faustinoaq** [comment]
We are using https://amberframework.gitbook.io/amber/getting-started now :tada: 

This ways is easier to read, maintain and contribute :sparkles: 

![screenshot_20180514_215014](https://user-images.githubusercontent.com/3067335/40034059-c990eb4e-57c0-11e8-930f-b9624fa35c4a.png)

I love the new and shiny `Edit on Github` button :100:  thanks to @GitbookIO :1st_place_medal:
